src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/BitConverterCompat.cs

```diff
@@ -0,0 +1,18 @@
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Data.SqlClient
+{
+    internal static class BitConverterCompat
```

Comment:
Did we truncate a word here/spelling mistake? Are you trying to say Compatible/Compact? I think it's okay to make it more clear if you need to say Compatible.

Reply:
@H-Yeo Yes, This was meant to be Compatible (since certain functions of bitconverter were not supported in older frameworks). I will make the necessary change .Thanks